.github/templates/linux_ci_workflow.yml.j2

```diff
@@ -11,8 +11,14 @@ on:
   pull_request:
 {%- endif %}
   push:
+{%- if enable_doc_jobs and is_scheduled %}
+    tags:
+      # NOTE: Binary build pipelines should only get triggered on release candidate builds
+      # Release candidate tags look like: v1.11.0-rc1
```

Comment:
The actual releases will not have the `-rc1` suffix. There is no need to trigger builds for them?